configure.ac

```diff
@@ -93,11 +93,18 @@ test -z "${tmpfilespath}" && tmpfilespath=/usr/lib/tmpfiles.d
 AC_SUBST(SYSTEMD_TMPFILESDIR, [${tmpfilespath}])
 
 AC_ARG_WITH([systemdsysctldir], AS_HELP_STRING([--with-systemdsysctldir=DIR],
-	    [path to systemd sysctl dir @<:@default=/usr/lib/sysctl.d@:>@]), [sysctlpath=${withval}],
-	    [sysctlpath="$($PKG_CONFIG --variable=sysctldir systemd)"])
+	    [path to systemd sysctl dir @<:@default=/usr/lib/sysctl.d@:>@]), [sysctlpath=${withval}])
 test -z "${sysctlpath}" && sysctlpath=/usr/lib/sysctl.d
 AC_SUBST(SYSTEMD_SYSCTLDIR, [${sysctlpath}])
 
+
+AC_ARG_WITH([systemdsystemctldir], AS_HELP_STRING([--with-systemdsystemctldir=DIR],
+            [path to systemd systemctl dir @<:@default=/usr/bin@:>@]), [systemctlpath=${withval}],
+            [systemctlpath="$($PKG_CONFIG --variable=systemctldir systemd)"])
```

Comment:
The `systemctldir` variable does not exist in systemd.pc, so you can omit the fourth argument to this macro.